utils/build-windows-toolchain.bat

```diff
@@ -201,8 +201,8 @@ cmake ^
   -D CMAKE_CXX_COMPILER=cl ^
   -D CMAKE_CXX_FLAGS="/GS- /Oy /Gw /Gy" ^
   -D CMAKE_MT=mt ^
-  -D CMAKE_Swift_COMPILER=%BuildRoot%/toolchains/swift-DEVELOPMENT-SNAPSHOT-2023-08-12-a/PFiles64/Swift/Toolchains/0.0.0+Asserts/usr/bin/swiftc.exe ^
```

Comment:
hmm, that `0.0.0+Asserts` might be load-bearing. I can't seem to find where `DEVTOOLS_ROOT` is specified in all of the XML, and I seem to have borked my Windows install so I can't just look at how it installs at the moment.

 https://github.com/apple/swift-installer-scripts/blob/097a839f0d6d310e2618f3894e2666c8ba622a90/platforms/Windows/bld/bld.wxs#L307

Reply:
Yeah, the 0.0.0+Asserts is the development builds.  5.9 did not get the new layout changes, so this should be correct.

Reply:
🤔 FWIW this is basically a cherry-pick of the 5.9 change so... weird.